rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -723,9 +750,10 @@ type TerminationProvision: <"A class for defining option provisions.">
     earlyTerminationProvision EarlyTerminationProvision (0..1) <"Parameters specifying provisions relating to the optional and mandatory early termination of a swap transaction.">
     evergreenProvision EvergreenProvision (0..1) <"A data defining: the right of a party to exercise an Evergreen option">
     extendibleProvision ExtendibleProvision (0..1) <"A provision that allows the specification of an embedded option with a swap giving the buyer of the option the right to extend the swap, in whole or in part, to the extended termination date.">
+    autocallableTermination KnockOutTermination (0..1) <"Early termination is triggered by the occurence of a Knock Out event.">
```

Comment:
Inconsistency between attribute name ('autocallable') and type name ('Knock-Out') is confusing. Pick one terminology and stick to it.